scene/gui/popup.cpp

```diff
@@ -112,7 +112,18 @@ void Popup::set_as_minsize() {
 	set_size(get_contents_minimum_size());
 }
 
+void Popup::set_close_on_parent_focus(bool p_close) {
+	close_on_parent_focus = p_close;
+}
+
+bool Popup::get_close_on_parent_focus() {
+	return close_on_parent_focus;
+}
+
 void Popup::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("set_close_on_parent_focus", "close"), &Popup::set_close_on_parent_focus);
```

Comment:
Why is this bound? It makes it appear in the documentation. Either remove the bind or make it a proper property with `ADD_PROPERTY` and fill the docs. Doesn't look necessary though.

Reply:
Yeah I thought I would expose it and just see what everyone thought about that, but it doesn't really need to be at all.